cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -578,6 +583,32 @@ def _add_pcluster_policies_to_role(self, role_ref: str, name: str):
             roles=[role_ref],
         )
 
+    def _add_custom_cookbook_policies_to_role(self, role_ref: str, name: str):
```

Comment:
why we can't add multiple resources to parallelcluster policy rather than create a new policy?
```
if self._condition_custom_cookbook_with_s3_url():
            resources=[
                self.format_arn(
                    service="s3",
                    resource="{0}-aws-parallelcluster/*".format(self.region),
                    region="",
                    account="",
                ),
             self.format_arn(
                    service="s3",
                    resource="{0}/*".format(bucket_name),
                    region="",
                    account="",
                )
            ],
  else:
            resources=[
              self.format_arn(
                  service="s3",
                  resource="{0}-aws-parallelcluster/*".format(self.region),
                  region="",
                  account="",
              ),
              ],
  
```

Reply:
I think we also create s3 policy for S3Access if there's S3Access. So I create new policy for customcook if there's s3 url to be aligned. Besides, if we add more policies in the parallelcluster policy in the futher, the if.. else will become longer and longer. So I think a new policy is more maintainable